en/docs/install-and-setup/manual.md

```diff
@@ -10,54 +10,42 @@
-4. Navigate to `<WSO2_OH_APIM_ACC_HOME>` directory.
-5. To setup the embedded H2 database of APIM with pre-loaded tables required by WSO2 Healthcare Solution, copy the `database/WSO2AM_DB.mv.db` file to `repository/database` folder in `<WSO2_APIM_HOME>`.
-
-    ### Optional
-    If you want create these tables in an existing installation of APIM, refer to the tables defined in the scripts under the `dbscripts` folder in `<WSO2_OH_APIM_ACC_HOME>`. You may have to manually execute the scripts in order to add them to an existing `WSO2AM_DB` database.
-
-6. In the `<WSO2_OH_APIM_ACC_HOME>` directory, execute 
+4. [Optional] Check the accelerator configurations in <WSO2_HC_APIM_ACC_HOME>/conf/config.toml file to enable or disable features.
+5. Navigate to `<WSO2_OH_APIM_ACC_HOME>` directory and execute following command. This will copy the artifacts to the WSO2 APIM and add the required configurations.
 ```sh
 ./bin/merge.sh
 ``` 
-command. This will copy the artifacts to the WSO2 APIM and add the required configurations.
 
-7. Navigate to `<WSO2_APIM_HOME>` directory and execute 
+6. Navigate to `<WSO2_APIM_HOME>` directory and execute following command to start the APIM server with WSO2 OH Accelerators.
 ```sh
 ./bin/api-manager.sh
 ```
-to start the APIM server with WSO2 OH Accelerators.
 
-### Audit Logs
-Running `./bin/merge.sh` script creates a audit log folder in the product home. Structure of it looks like below;
+???+ note
+    Running `./bin/merge.sh` script creates a audit log folder in the product home. Structure of it looks like below;
 
+    ```
+    hc-accelerator
+    ├── backup
+    │   ├── conf
+    │   ├── jaggeryapps
+    │   └── webapps
+    └── merge_audit.log
 
-```
-oh-accelerator
-├── backup
-│   ├── conf
-│   ├── jaggeryapps
-│   └── webapps
-└── merge_audit.log
-
-```
+    ```
 
-- `merge_audit.log` will have an audit line per execution of the `merge.sh` script of the accelerator. Each line contains execution date and time, user account and the version of the accelerator. Example log line is below;
-```buttonless
-Mon May 31 22:01:55 +0530 2021 - john - WSO2 Healthcare Solution API Manager 4.0.0 Accelerator - v3.0.0
-```
-- `backup` folder contains the files that were originally there in the APIM product before running the accelerator. Please note that only the last state will be there. 
+    - `merge_audit.log` will have an audit line per execution of the `merge.sh` script of the accelerator. Each line contains execution date and time, user account and the version of the accelerator. Example log line is below;
+    ```buttonless
+    Mon May 31 22:01:55 +0530 2021 - john - WSO2 Healthcare Solution API Manager 4.0.0 Accelerator - v3.0.0
+    ```
+    - `backup` folder contains the files that were originally there in the APIM product before running the accelerator. Please note that only the last state will be there. 
 
 ### Product Compatibilities
 
 | Product          | Compatible Open Healthcare Accelarator  |
```

Comment:
need to refactor Open Healthcare into Healthcare Solution
https://github.com/wso2-enterprise/open-healthcare/issues/1653